resources/web/wwi/animation.js

```diff
@@ -110,14 +122,14 @@ class Animation { // eslint-disable-line no-unused-vars
   }
 
   _updateSlider(value) {
-    var clampedValued = Math.min(value, 99); // set maximum value to get valid step index
-    var requestedStep = Math.floor(this.data.frames.length * clampedValued / 100);
+    const clampedValued = Math.min(value, 99); // set maximum value to get valid step index
+    const requestedStep = Math.floor(this.data.frames.length * clampedValued / 100);
     this.start = (new Date().getTime()) - Math.floor(this.data.basicTimeStep * this.step);
     this._updateAnimationState(requestedStep);
   }
 
-  _updateAnimationState(requestedStep = undefined) {
-    var automaticMove = typeof requestedStep === 'undefined';
+  async _updateAnimationState(requestedStep = undefined) {
```

Comment:
It should not be async as it doesn't call async function?

Reply:
fixed in 42914ba